internal/pkg/task/env_runner.go

```diff
@@ -156,3 +157,48 @@ func containsString(s []string, search string) bool {
 	}
 	return false
 }
+
+// CheckNonZeroExitCode returns the status of the containers part of the given tasks.
+func (r *EnvRunner) CheckNonZeroExitCode(tasks []*Task) error {
```

Comment:
wonder if there'd be a good way to call a common function between this one and the one in `config_runner`🤔, it looks pretty similar